ibis/backends/tests/test_temporal.py

```diff
@@ -1000,79 +982,50 @@ def test_temporal_binop(backend, con, alltypes, df, expr_fn, expected_fn):
                     raises=com.IbisTypeError,
                     reason="Given argument with datatype interval('D') is not implicitly castable to string",
                 ),
-                pytest.mark.notimpl(
-                    ["bigquery"],
-                    raises=com.UnsupportedOperationError,
-                    reason='BigQuery does not allow binary operation TIMESTAMP_ADD with INTERVAL offset D',
-                ),
-            ],
-        ),
-        param(
-            '1.5d',
-            plus,
-            marks=[
-                pytest.mark.broken(["mysql"], raises=AssertionError),
-                pytest.mark.broken(
-                    ['druid'],
-                    raises=com.IbisTypeError,
-                    reason="Given argument with datatype interval('s') is not implicitly castable to string",
-                ),
-                pytest.mark.broken(
-                    ["bigquery"],
-                    raises=GoogleBadRequest,
-                    reason='400 Syntax error: Expected ")" but got integer literal "12" at [1:58]',
-                ),
             ],
         ),
+        # param(
+        #     '1.5d',
```

Comment:
Currently raising `ValueError: Cannot normalize timedelta with days and seconds or microseconds`